packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -16172,8 +16172,134 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /events/attributes:
+    get:
+      tags:
+        - events
+      summary: Get Attribute events
+      description: |
+        Retrieve list of Attribute events that occurred after the specified event ID.
+        Results are ordered in ascending order (oldest events first).
+      operationId: getAttributeM2MEvents
+      parameters:
```

For these new endpoints, I would consider reusing the pattern we used in the m2mEventApi, where we defined reusable parameters and used references. Something like:

```suggestion
      parameters:
        - $ref: "#/components/parameters/LastEventId"
        - $ref: "#/components/parameters/EventsLimit"
```